### PR TITLE
CF-605 Fix cloudferry_devlab reqiurements: test-generator.

### PR DESCRIPTION
**Pull request by plomakin**
Fixed test-generator dependecy.

### CODE REVIEW

File: cloudferry_devlab/requirements.txt [hunk 1]
```diff
@@ -14,4 +14,4 @@ nose==1.3.7
 nose-testconfig==0.9.1
 sqlalchemy==1.0.12
 PyMySQL==0.6.7
--e git+https://github.com/kevinastone/generator.git@nose-attribs#egg=test-generator
+test-generator
```
Comment (SVilgelm) [inline cloudferry_devlab/requirements.txt]: version?

**Comment by SVilgelm** [comment]
👎

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request.

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request.

**Comment by roman-verchikov** [comment]
+1

**Comment by SVilgelm** [comment]
👍

**Pull request merged**